installer/cmd/render.go

```diff
@@ -40,10 +41,15 @@ A config file is required which can be generated with the init command.`,
   gitpod-installer render --config config.yaml --namespace gitpod | kubectl apply -f -`,
 	RunE: func(cmd *cobra.Command, args []string) error {
 		cfgFN, _ := cmd.PersistentFlags().GetString("config")
-		cfg, err := config.Load(cfgFN)
+
+		rawCfg, cfgVersion, err := config.Load(cfgFN)
 		if err != nil {
 			return fmt.Errorf("error loading config: %w", err)
 		}
+		if cfgVersion != config.CurrentVersion {
+			return fmt.Errorf("config version is too old: expected %s, got %s", config.CurrentVersion, cfgVersion)
```

Comment:
how about, `config version mismatch: expected %s, got %s`. User might try newer version of config with old version of installer.

Reply:
Excellent point. I've updated this.